### ISSUE TITLE
Improve getting started guide

### ISSUE THREAD

**Issue by vthunder** [mozilla/persona#3083]
Hello!

I'm trying out this Persona thing I've heard so much about ;-) Just wanted to give my $.02 on the getting started guide, since I just went through it with fresh eyes.
- Too wordy

The best guides out there have only just enough words to let you understand what the bits of code you need to paste into your terminal/editor do. Leave the details for a reference section of the docs. I'd say the wordage can easily be cut in 1/2.
- It should have you put an icon on the page

Literally, put in a line to copy and paste which is like `<img id="signin"...>`, and have me do that _before_ giving me the JS that attaches to the buttons. Don't have me navigate to another page to get the buttons.
- Copying and pasting code should not trigger the "infinite refresh" loop

The code provided hard-codes "bob@example.com" as loggedInUser, so when you copy and paste that code, and try it out, you get stuck in an infinite refresh loop. Not sure how exactly the copy & paste code needs to get around that, but it needs to be harder to end up in this broken state.
- Should probably have a link to the cookbook:

https://github.com/mozilla/browserid-cookbook

Maybe even at the top.
- Maybe code for async loading of the JS library. If this is not desirable, maybe a mention for what to search for to get help in implementing that.

Hope this helps!

**Comment by callahad** [comment]
Hi! To help us better focus, I'm "closing" all issues that have been open for more than six months. These have been tagged "cleanup-2014" so that we can go back and review them in the future.

For more information, check out this thread: http://thread.gmane.org/gmane.comp.mozilla.identity.devel/7394

If you believe this bug is still a major issue for you, please comment, submit a pull request, or discuss it on our mailing list: https://lists.mozilla.org/listinfo/dev-identity

Sorry for GitHub notification spam!
